common/test/acceptance/pages/lms/instructor_dashboard.py

```diff
@@ -880,6 +886,44 @@ def get_notification_text(self):
         return self.q(css="{} h3".format(notification_selector)).text
 
 
+class MembershipPageBetaTesterSection(PageObject):
+    """
+    Beta tester section of the Membership tab of the Instructor dashboard.
+    """
+    url = None
+
+    batch_beta_tester_heading_selector = '#heading-batch-beta-testers'
+    batch_beta_tester_selector = '.batch-beta-testers'
+
+    def is_browser_on_page(self):
+        return self.q(css=self.batch_beta_tester_heading_selector).present
+
+    def fill_beta_tester_text_box(self, username):
```

Comment:
Because of this `is_browser_on_page `. So what's your suggestion if we changes the name?